client/starwhale/api/_impl/data_store.py

```diff
@@ -575,7 +577,7 @@ def get_instance() -> "LocalDataStore":
 
     def __init__(self, root_path: str) -> None:
         self.root_path = root_path
-        self.name_pattern = re.compile(r"^[A-Za-z0-9-_/]+$")
+        self.name_pattern = re.compile(r"^[A-Za-z0-9-_/ ]+$")
```

Comment:
use `starwhale.utils.validate_obj_name`  to check object validation?  [code link](https://github.com/star-whale/starwhale/blob/main/client/starwhale/utils/__init__.py#L122)

another comment: why support blank?

Reply:
because some metric have blank，such as ：_summary_m = ["accuracy", "macro avg", "weighted avg"]